tests/integration/session/test_timeout.py

```diff
@@ -32,12 +30,3 @@ def test_when_session_times_out_server_side_401_is_returned(self):
         time.sleep(3)
         self.get(self.last_url)
         self.assertStatusUnauthorised()
-
-    def test_schema_defined_timeout_is_used(self):
-        self._application.config['EQ_SESSION_TIMEOUT_SECONDS'] = 10
-        self.launchSurvey('test', 'timeout')
-        self.assertInHead('window.__EQ_SESSION_TIMEOUT__ = 6')
-
-    def test_schema_defined_timeout_cant_be_higher_than_server(self):
```

Comment:
Quick point, it is still worth keeping this check. 
*Ensuring schema defined timeout cannot be longer than the app default is useful.*

Reply:
As far as I understand, this test only ensures that `window.__EQ_SESSION_TIMEOUT__ = 2` is correctly set which has now been removed in `/app/templates/layouts/_base.html`.

Reply:
I guess a way of still achieving the same result would be to set `'EQ_SESSION_TIMEOUT_SECONDS'` lower than the schema timeout. Then you can `self.assertStatusUnauthorised()` after the sleeping for app specified timeout. This ensure it used the app value and not the higher schema value.

Reply:
So at the moment during the test setUp the system timeout is being set to two seconds. Previously the schema had a timeout of 6 seconds which is greater than the system setting of two. Therefore the test was checking that  it used the system (2) rather than schema defined (6) timeout. This should therefore stay but it might be worth either decreasing the system defined timeout to 1 second in the setUp() or increasing the schema defined timeout to at least 3.